styx-cli/src/main/java/com/spotify/styx/cli/Main.java

```diff
@@ -562,7 +571,7 @@ public Subparser parser(Subparsers subCommands) {
   }
 
   private enum BackfillCommand {
-    LIST("ls", "List backfills"),
+    LIST("ls", "List backfills, by default, only active backfills are printed. Use option -a (--show-all) to show all"),
```

Comment:
A bit too long shown in the terminal. How about `List active backfills. Use option -a (--show-all) to show all`? Or we just rely user to do `styx backfill list -h` and just say `List active backfills`